src/Analyzer.Core/TemplateAnalyzer.cs

```diff
@@ -76,10 +76,9 @@ public static TemplateAnalyzer Create(bool usePowerShell, ILogger logger = null)
         /// <summary>
         /// Runs the TemplateAnalyzer logic given the template and parameters passed to it.
         /// </summary>
-        /// <param name="template">The ARM Template JSON</param>
-        /// <param name="parameters">The parameters for the ARM Template JSON</param>
-        /// <param name="templateFilePath">The ARM Template file path. (Needed to run arm-ttk checks.)</param>
-        /// <returns>An enumerable of TemplateAnalyzer evaluations.</returns>
+        /// <param name="template">The template contents.</param>
+        /// <param name="parameters">The parameters for the template.</param>
+        /// <param name="templateFilePath">The template file path.</param>
```

Comment:
I would leave the "(Needed to run arm-ttk checks.)" if that is how it still works

Reply:
It's not true, now Bicep also needs the file path